core/Cargo.toml

```diff
@@ -18,14 +18,23 @@ categories = ["no-std", "embedded"]
 include = ["/Cargo.toml", "src/**/*.rs", "/README.md", "/LICENSE"]
 
 [dependencies]
-ink_alloc = { path = "../alloc/" }
-ink_utils = { path = "../utils/" }
-parity-codec = { version = "3.2", default-features = false, features = ["derive", "full"] }
+ink_abi = { path = "../abi", default-features = false }
+ink_alloc = { path = "../alloc/", default-features = false }
+ink_utils = { path = "../utils/", default-features = false }
+parity-codec = { version = "4.1", default-features = false, features = ["derive", "full"] }
+
+[dependencies.type-metadata]
+git = "https://github.com/type-metadata/type-metadata.git"
+default-features = false
+features = ["derive"]
```

Comment:
this is for your abi PR?

Reply:
damn, I somehow accidentally based the initial PR on ink-abi branch and git was very unforgiving about that. thanks